gui/src/app/SPAnalysis/SPAnalysisQueryLoading.ts

```diff
@@ -65,20 +70,36 @@ const deepCopy = (obj: any) => {
 }
 
 export const fetchRemoteAnalysis = async (query: QueryParams) => {
-    // any special 'project' query could be loaded here at the top
-    const data: SPAnalysisDataModel = deepCopy(initialDataModel)
+    const projectUri = query.project
+
+    let data: SPAnalysisDataModel = deepCopy(initialDataModel);
+    if (projectUri) {
+        if (projectUri.startsWith('https://gist.github.com/')) {
+            const contentLoadedFromGist = await loadFilesFromGist(projectUri);
```

Comment:
Even if we don't do anything in the `catch` for now, we should probably wrap this call in a try - right now it just logs an error to the console and prevents the rest of the hook that calls it from running

Reply:
Do we want to allow it to continue and load the other parts of the query (if they are present) even if the main project load failed? I was thinking we could catch the error and put an alert popup that project couldn't load and then return.

Reply:
I don't feel strongly either way, if other things are provided and they succeed that seems fine to me. An alert seems like the right thing to do, but anything is probably better than propagating the exception

Reply:
There's now a try/catch and an alert message; the rest of the loading skipped. I feel this is good for now.